include/net/net_pkt.h

```diff
@@ -2070,9 +2070,9 @@ struct net_pkt *net_pkt_rx_alloc_with_buffer_debug(struct net_if *iface,
 #define net_pkt_rx_alloc_with_buffer(_iface, _size, _family,		\
 				     _proto, _timeout)			\
 	net_pkt_rx_alloc_with_buffer_debug(_iface, _size, _family,	\
-					      _proto, _timeout,		\
-					      __func__, __LINE__)
-#endif
+					   _proto, _timeout,		\
+					   __func__, __LINE__)
+#else /* NET_PKT_DEBUG_ENABLED */
 /** @endcond */
```

Comment:
The `@endcond` looks a bit suspicious now as it is after the else statement. The individual #ifdefs were there on purpose so that the documentation is properly generated. Can you rework this commit and only do the indentation issue instead.

Reply:
yep noticed that #ifdef have to be per-function due to doc generation. too bad